build/build-docker-deploy.sh

```diff
@@ -2,7 +2,4 @@
 
 set -eu
 
-cd "$(dirname $0)/.."
-
-./build/build-docker-dev.sh
-./build/deploy/mkimage.sh
+$(dirname $0)/deploy/mkimage.sh
```

Comment:
This is done. I've moved the contents of `deploy/mkimage.sh` into `build/build-docker-deploy.sh`.